### DISCUSSION TITLE
Unknown electricity source

### DISCUSSION THREAD

**Discussion by charlottejungen** [electricitymaps/electricitymaps-contrib#6956] [Q&A] [answered]
Hi,

Does someone know how the electricity sources labelled as 'unknown' are set up and why is it defined as unknown?

Thanks in advance for helping!

Charlotte

**Comment by madsnedergaard** [comment, marked as answer]
Hi @charlottejungen, thanks for asking.

"Unknown" basically means we don't know what production mode was used to generate the electricity, but we know that this specific amount was generated. 

For many zones, this is thermal (coal, gas, oil or biomass) as those numbers are not necessarily reported on an hourly level. 

In the tooltip we sometimes are able to explain more precisely what "unknown" entails for a specific zone, see example from Spain here:

![Screenshot 2024-07-15 at 15 00 24](https://github.com/user-attachments/assets/0ce001f6-1410-4cec-b670-7528fbf0db62)

This is good feedback that we should make this more clear in the app though, so thanks for bringing it up :)

**Comment by charlottejungen** [comment]
Hi Mads,

Thank you for your respond. I would really like to use the free data available on your website to calculate carbon footprints or impacts, however is it hard to define a emission factor when there is electricity from sources that are unknown. Do you have any recommendations on how I can still incorporate this data into my calculations? Is there like a environmental factor that can be estimated for the unknown part by for example taking the average of the emission factors of the most common electricity sources that fall under 'unknown'?

Thanks in advance for your time and help.

Kind regards,

Charlotte
________________________________

**Reply by VIKTORVAV99** [reply]
There is no environmental factor that can be applied to all unknown data as this varies not only by region or zone but also has a seasonal factor to it where it changes depending on the season. There is also a trending change historically in most cases where it either increases or decreases over time which mean that the production ratios from unknown and therefore the emissions changes.

We have chosen to take a weighted average of the yearly unknown production where we have yearly sources with that breaks down the unknown hourly data. But even this has its drawbacks as it misses on the seasonality and the trending historical changes are delayed by roughly 2-3 years. 


If you however have a single zone in mind it might be possible for you or your team to create a somewhat accurate breakdown of the unknown data for that zone taking seasonality and trending production into account.

**Comment by charlottejungen** [comment]
Thank you for your clear explanation!

You have chosen to take a weighted average of the yearly unknown productions. For example, I often see the 700 gCO2eq/kWh that is being used and consists of coal, gas, oil or biomass. Do you perhaps have the ratio of these sources that is being used for this value? I am just trying to understand the accuracy of the unknown.

Next, you explained that it might be possible for a single zone to breakdown the unknown data. I am specifically interested in the Netherlands, however that 'Unknown 'does not state where it consists of (in comparison to for example the 700 gCO2eq/kWh that is used for a lot of countries their unknown).
Do you perhaps know how to get a general breakdown of the sources in unknown for the Netherlands. For example the ratio of the sources that are used that result in the 342 gCO2/eq/kWh? I saw that it is based on the Electricity Maps 2020 data, but I can't find it there.

Thanks in advance for your help!


________________________________

**Reply by VIKTORVAV99** [reply]
We are currently looking into Netherlands ourselves and will offer a more accurate carbon intensity figure when we can. We have currently reached out to ENTSO-E and TENNET NL to try and get a better understanding of what they report as unknown.

Right now it's using the global fallback of 700 for unknown.

The first step will be to simply adjust the unknown carbon intensity based on the information we get from either ENTSO-E or TENNET.

Depending on how static or dynamic it is we might then choose to do an estimation model that directly breaks down the unknown data into the various production modes. But that's not something I can promise we will do as it depends on what data we can get to potentially build such an estimation model.

**Comment by charlottejungen** [comment]
Ok! You mean it is now using the 700 gCO2eq/kWh value? Because I see 342 gCO2eq/kWh for the Netherlands.
Hopefully TENNET or either ENTSO-E will provide you with some useful data soon 😄.

Do you maybe have a ratio of the coal, gas, oil or biomass that is used to calculate the 700 gCO2eq/kg for the global fallback number of the unknown? That would be very helpful information.

Thanks in advance!
________________________________

**Reply by VIKTORVAV99** [reply]
My bad we are indeed using 342 for Netherlands. I did not know it had already been adjusted.

The global fallback was something that was chosen a long time ago based on the available data at the time. I do not have a exact ratio that was used for that.

**Reply by VIKTORVAV99** [reply]
I just wanted to let you know we identified a problem with double accounting solar production. This as now been fixed and we are refetching all the data and reprocessing it which should be all done by the end of the week. (Might take until next the yearly data in the app has changed due to caching).

The basic problem is that the Solar reported to ENTSO-E is super low so we decided to get this from NED.nl instead but it turns out that after roughly I month I think it was the unknown data in ENTSO-E got a massive increase, most of which can be attributed to Solar in our investigation.

**Comment by charlottejungen** [comment]
I tried to get an average of 700 gCO2e/kWh by combining the GWP per kWh from ecoinvent considering coal, gas, oil or biomass, but noticed that the combination is not able to give a value of 700 grams because all individual contributions are much higher than 700 g ( ~ 2 kg for oil, 1 kg for coal, 1 kg for biomass and 8 for natural gas). Do you perhaps know why this data is so much higher compared to your calculations? Both Electricity maps and Ecoinvent use the IPCC 2014 emission factor.

________________________________

**Comment by charlottejungen** [comment]
Thank you for your message. That is interesting and would make sense. Is there a possibility that when the new value is calculated, it can be shared on GitHub even though it will not be visible in the app yet? That would help a lot with making calculations and maybe a better insight in the sources taking into account in that value, because the app does not say it contains solar.

Thanks,
Charlotte
________________________________

**Reply by VIKTORVAV99** [reply]
I don't think we will be able to share this on GitHub simply because the amount of data, it will be hourly values of every hour going back to 2017.

We are not adjusting the unknown carbon intensity yet until we figure out what the remainder consists and we are looking into further reducing the unknown values by looking at different data sources. Right now we are looking at incorporating more data from NED.nl for more than solar but we have not had time to evaluate them fully yet.
